tests/playbooks/tests_team.yml

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: team0
+    profile: "{{ interface }}"
+    lsr_fail_debug:
+      - __network_connections_result
+  tasks:
+    - debug:
+        msg: "this is: playbooks/tests_team.yml"
+      tags:
+        - always
+
+    - block:
+        - include_tasks: tasks/run_test.yml
+          vars:
+            lsr_description: I can create a profile
+            lsr_setup:
+              - tasks/delete_interface.yml
+              - tasks/assert_device_absent.yml
+            lsr_test:
+              - tasks/create_team_profile.yml
+            lsr_assert:
+              - tasks/assert_profile_present.yml
```

Comment:
Should this also assert that the interface/device was created?

Reply:
updated